systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -223,8 +229,7 @@
     String EXTERNAL_CLIENTS_USED = "externalclients";
 
     /**
-     * Constants for Kafka clients labels
+     * Tag for tests running on Azure Pipelines
      */
-    String KAFKA_CLIENTS_LABEL_KEY = "user-test-app";
-    String KAFKA_CLIENTS_LABEL_VALUE = "kafka-clients";
+    String AZURE = "azure";
```

Comment:
Would we really define which tests to run on the Azure CI like this? Or would it be some set of tests selected via other tags? Even if it were explicit using a new tag, should we call it CI, rather making it about Azure itself, which is merely the platform which we're using to run the tests.

Reply:
Create a new tag is a simple thing. We cannot run there whole regression tests, because HW limitations for azure VM will cause several failures. We can maybe use `acceptance` tag and slightly extend it with some cases which are not currently under `azure` tag.